package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@segment/analytics.js-integration-amplitude",
   "description": "The Amplitude analytics.js integration.",
-  "version": "2.4.1",
+  "version": "2.5.0",
```

Comment:
wondering if all this new feature behavior be a major release?

Reply:
Yeah but it's all still backwards compatible (if we're following SEMVER that's not major)

Reply:
sounds good!